src/wap_client.c

```diff
@@ -177,7 +177,7 @@ prepare_get_command (client_t *self)
 static void
 signal_have_get_ok (client_t *self)
 {
-    zsock_send (self->cmdpipe, "sip", "GET OK", 0, 
+    zsock_send (self->cmdpipe, "sip", "GET OK", 0,
                 wap_proto_get_tx_data (self->message));
```

Comment:
Whitespace change, not significant.

On Sun, Jan 4, 2015 at 4:27 PM, oranjuice <notifications@github.com> wrote:

> In src/wap_client.c
> <https://github.com/oranjuice/monero-0mq/pull/4#discussion-diff-22439996>:
>
> > @@ -177,7 +177,7 @@ prepare_get_command (client_t *self)
> >  static void
> >  signal_have_get_ok (client_t *self)
> >  {
> > -    zsock_send (self->cmdpipe, "sip", "GET OK", 0,
> > +    zsock_send (self->cmdpipe, "sip", "GET OK", 0,
> >                  wap_proto_get_tx_data (self->message));
>
> How does this change relate to the "start" command whoss behaviour we're
> trying to change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oranjuice/monero-0mq/pull/4/files#r22439996>.
>